src/utils/xml_helpers.py

```
@@ -1,32 +1,37 @@
 import xml.etree.ElementTree as ET
 from datetime import datetime
 from typing import Dict, List
-from models.feed import Feed
+from ..models.feed import Feed
 
-def create_opml_tree(feeds_by_genre: Dict[str, List[Feed]], title: str = "RSS Feeds") -> ET.Element:
+
+def create_opml_tree(
+    feeds_by_genre: Dict[str, List[Feed]], title: str = "RSS Feeds"
```

Ruff (0.8.0)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)